portfolio/src/main/webapp/script.js

```diff
@@ -129,4 +129,12 @@ function getComments(maxComments) {
         commentsHistory.appendChild(listNode);
     })
   });
+}
+
+/** Deletes Comments */
+function deleteComments() {
+  const request = new Request("/delete-comments", {method: 'POST'});
+  fetch(request).then(() => {
+    getComments(0);
```

Comment:
I think you're calling `getComments(0)` in your response callback here so that the comments section of the page gets updated, which is good. However, I think that asking for zero comments could backfire; imagine a bug is introduced such that the backend doesn't delete all comments, but still returns an OK response. The frontend would turn around and fetch zero comments, and it would look to the user like there was a success. However, the remaining comments would reappear upon page reload.

I think the hypothetical I described is unlikely, but still I think it would be better to fetch comments with the same limit as normal, in case something weird happens; it's better for the user to notice a problem right away.

Reply:
Ah, I hadn't thought of that aspect! I'll change this to avoid this potential issue, thanks!